### ISSUE TITLE
Make information about contacts less redundant in the location form (Archive & Delete)

### ISSUE THREAD

**Issue by juliankehne** [digitalfabrik/integreat-cms#3147]
### Motivation
Duplicates of contacts and information texts are redundant in the "Actions" tab of a Location and take up a lot of space. 
Archive and Delete options each have a copy of all linked contacts.

If one location has a lot of contacts the Actions tab will include a lot of contact entries.

See Screenshot.

### Proposed Solution

Solution 1: Have only one list of linked contacts

Solution 2: Solution 1 + The texts "You cannot archive a location which is referenced by a contact.
To archive this location, you have to delete these contacts first: " and for deleting seem redundant as well and could be merged into one message to:

"You cannot delete or archive a location..."

### Alternatives
Keep as it is or @nikolahoff can probably come up with something better :)

### User Story
As a manager I want user interface without a lot of redundancy to quickly see which contacts are blocking the deletion of a location / archiving of a location.

### Additional Context

![2024-10-23 13_13_05-Window](https://github.com/user-attachments/assets/a5d82d2a-1126-414f-b63e-70d1f9e02a65)

**Comment by osmers** [comment]
I guess for users in municipalities this should generally not present a problem, bcs they cannot delete. Nonetheless, we can think about how to optimize it for us :)

**Comment by osmers** [comment]
Maybe we can also address the issue, that pois which are referenced by events and contacts have the same text twice: 
![image](https://github.com/user-attachments/assets/100a20dd-bdb5-46a1-9c09-d835357f2e38)
We can probably combine that as well, e.g. by writing one warning text and then subdivide into events and contacts using headings.

**Comment by MizukiTemma** [comment]
After #3087 there will be a list of contacts which are referring the POI opened in the form. It looks like the first photo currently:

![Screenshot 2024-11-05 183515](https://github.com/user-attachments/assets/5d750e70-89dd-4d75-bad3-63fed9f200b7)

while the action box looks like this in the same example:
![Screenshot 2024-11-05 183540](https://github.com/user-attachments/assets/c3c52497-46c8-42c6-844c-90e8d9095bec)

**Comment by nikolahoff** [comment]
@osmers if this is still open we can discuss it in our UX <> Dev meeting.

**Comment by nikolahoff** [comment]
@osmers @MizukiTemma [here](https://www.figma.com/design/6U7R7Xj4wL7sbjxKRmOG9D/CMS-Project?node-id=2803-362&t=bs9cXpjHQuUHs0r0-1) ist the design for the updated action tile.

**Comment by osmers** [comment]
@nikolahoff is that the right file? It take me to the new feedback design...

**Comment by nikolahoff** [comment]
@osmers sorry I updated it above

**Comment by nikolahoff** [comment]
@MizukiTemma the design is done, you can find it [here](https://www.figma.com/design/6U7R7Xj4wL7sbjxKRmOG9D/CMS-Project?node-id=2819-10537&t=YtR7uj26Rdsihour-1).

**Comment by osmers** [comment]
I added a screenshot of the design - do we need separate designs for all instances (Deleting/archiving POI, that is referenced by event/contact)?

**Comment by MizukiTemma** [comment]
No, we can implement in the same/similar way with the design for other content types too.

**Comment by osmers** [comment]
@m0ndra I would still ask you to maybe have a look at this and put the design into your format. We can get together to look at the context of this issue if something is not clear to you :)
